templates/zerver/api/outgoing-webhooks.md

```diff
@@ -107,6 +107,10 @@ encode a JSON dictionary that contains `response_not_required` set
 to `True`, so that no response message is sent to the user.  (This
 is helpful to distinguish deliberate non-responses from bugs.)
 
+### Return Values
+
+{generate_return_values_table|zulip.yaml|/zulip-outgoing-webhook:post}
```

Comment:
I think this needs a different title and should mostly replace the whole `Zulip message format` section, after perhaps moving some useful content from that section to the `zulip-outgoing-webhook` docs, right?  

Also, do we validate the OpenAPI data in our backend unit tests?  We might need to add some logic in `zerver/tests/test_outgoing_webhook_*` to call `validate_against_openapi_schema` or something.